src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppAppRunner.recipe

```diff
@@ -0,0 +1,285 @@
+                "1024",
+                "2048"
+            ],
+            "ValueMapping": {
+                "1024": "1 vCPU",
+                "2048": "2 vCPU"
+            }
+        },
+        {
+            "Id": "Memory",
+            "Name": "Memory",
+            "Description": "The amount of memory reserved for each instance of your App Runner service.",
+            "Type": "String",
+            "AdvancedSetting": false,
+            "Updatable": true,
+            "DefaultValue": "2048",
+            "AllowedValues": [
+                "2048",
+                "3072",
+                "4096"
+            ],
+            "ValueMapping": {
+                "2048": "2 GB",
+                "3072": "3 GB",
+                "4096": "4 GB"
+            }
+        },
+        {
+            "Id": "EncryptionKmsKey",
+            "Name": "Encryption KMS Key",
+            "Description": "The ARN of the KMS key that's used for encryption of application logs.",
+            "Type": "String",
+            "AdvancedSetting": true,
+            "Updatable": false
+        },
+        {
+            "Id": "HealthCheckProtocol",
+            "Name": "Health Check Protocol",
+            "Description": "The IP protocol that App Runner uses to perform health checks for your service.",
+            "Type": "String",
+            "DefaultValue": "TCP",
+            "AdvancedSetting": true,
+            "Updatable": true,
+            "AllowedValues": [
+                "HTTP",
+                "TCP"
+            ]
+        },
+        {
+            "Id": "HealthCheckPath",
+            "Name": "Health Check Path",
+            "Description": "The URL that health check requests are sent to.",
+            "Type": "String",
+            "AdvancedSetting": true,
+            "Updatable": true,
+            "DependsOn": [
+                {
+                    "Id": "HealthCheckProtocol",
+                    "Value": "HTTP"
+                }
+            ]
+        },
+        {
+            "Id": "HealthCheckInterval",
+            "Name": "Health Check Interval",
+            "Description": "The time interval, in seconds, between health checks.",
+            "Type": "Int",
+            "DefaultValue": "5",
+            "AdvancedSetting": true,
+            "Updatable": true
+        },
+        {
+            "Id": "HealthCheckTimeout",
+            "Name": "Health Check Timeout",
+            "Description": "The time, in seconds, to wait for a health check response before deciding it failed.",
+            "Type": "Int",
+            "DefaultValue": "2",
+            "AdvancedSetting": true,
+            "Updatable": true
+        },
+        {
+            "Id": "HealthCheckHealthyThreshold",
+            "Name": "Health Check Healthy Threshold",
+            "Description": "The number of consecutive checks that must succeed before App Runner decides that the service is healthy.",
+            "Type": "Int",
+            "DefaultValue": "3",
```

Comment:
nit: remove quotes from 3 to indicate it's an Int

Reply:
Done